src/options/smt_options.toml

```diff
@@ -342,6 +342,14 @@ header = "options/smt_options.h"
   type       = "bool"
   default    = "false"
   help       = "calculate sort inference of input problem, convert the input based on monotonic sorts"
+  
+[[option]]
+  name       = "symmetryBreaker"
+  category   = "regular"
+  long       = "symmetry-break"
+  type       = "bool"
+  default    = "false"
+  help       = "generate symmetry breaking constraints after the symmetry detection and append them to the original assertions"    
```

Comment:
This option should probably automatically enable `--symmetry-detect`, too. Also, do we need those to be separate options?

Reply:
I agree we don't need separate options for this.

Reply:
I removed --symmetry-detect and only use symmetry-breaker-exp to enable the symmetry breaking.